Fix two page layout screenshot failures

## Description

Since  #2141, we have a flag in `pytest` to generate automatic screenshots for each page of the source and journalist interface in each translated language. There is a previously unused step (`_admin_creates_a_user`) in the `functional/journalist_navigation_steps.py` that are used by these tests of the page layout. Two of these screenshots are failing (details below).

Note that this was caused by merging in #1509 and #2141 in quick succession. 

## Steps to Reproduce

1. `vagrant up`
2. `cd /vagrant/securedrop`
3. `pytest -v tests/ --page-layout`

## Expected Behavior

All tests pass (and thus all screenshots are generated)

## Actual Behavior

Everything except `test_admin_new_user_two_factor_hotp` and `test_admin_new_user_two_factor_totp` pass. One sees errors like the following:

```
self = <test_journalist.TestJournalistLayout instance at 0x7f32080daa28>

    def test_admin_new_user_two_factor_hotp(self):
        self._admin_logs_in()
        self._admin_visits_admin_interface()
>       self._admin_creates_a_user(hotp='123456')

tests/pages-layout/test_journalist.py:96: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <test_journalist.TestJournalistLayout instance at 0x7f32080daa28>, hotp = '123456'

    def _admin_creates_a_user(self, hotp):
        add_user_btn = self.driver.find_element_by_css_selector(
            'button#add-user')
        add_user_btn.click()
    
        self.new_user = dict(
            username='dellsberg',
            password='pentagonpapers')
    
        import pdb; pdb.set_trace()
        self._add_user(self.new_user['username'],
                       self.new_user['password'],
                       is_admin=False,
>                      hotp=hotp)
E       TypeError: _add_user() got multiple values for keyword argument 'is_admin'

tests/functional/journalist_navigation_steps.py:521: TypeError
```

## Comments

First I'm going to add these page-layout screenshots to CI so that if this happens again it'll fail the build (#2181). 

Then I'll fix the underlying cause here, which is the following code stanza in `_admin_creates_a_user` still has a reference to the password that we're now auto-generating for all users:

```py
        self._add_user(self.new_user['username'],
                       self.new_user['password'],
                       is_admin=False,
                       hotp=hotp)
```

Removing `self.new_user['password']` fixes.

--- a/.travis.yml
+++ b/.travis.yml
@@ -51,7 +51,7 @@ script:
   # Travis needs the config.py file to be owned by root. In other environments
   # it's the `securedrop_user` var.
   - sudo chown root:root securedrop/config.py
-  - sudo sh -c "export DISPLAY=:1; cd securedrop/ && pytest -v tests/"
+  - sudo sh -c "export DISPLAY=:1; cd securedrop/ && pytest -v tests/ --page-layout"
   - SECUREDROP_TESTINFRA_TARGET_HOST=travis testinfra -v testinfra/development/
   - pip freeze -l
   # Performing new pip install step *after* build VM tests pass, so as not

--- a/securedrop/tests/functional/journalist_navigation_steps.py
+++ b/securedrop/tests/functional/journalist_navigation_steps.py
@@ -532,7 +532,6 @@ class JournalistNavigationSteps():
             password='pentagonpapers')
 
         self._add_user(self.new_user['username'],
-                       self.new_user['password'],
                        is_admin=False,
                        hotp=hotp)
 

